domains/win64.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "userwin64,
+        "email": "win64inquiries@gmail.com"
+    },
+    "record": {
+        "CNAME": "win64.is-a.dev"
```

Comment:
Invalid record, same as the domain you are requesting. Please read the docs before making a PR: https://docs.is-a.dev